dask_deltatable/core.py

```diff
@@ -341,6 +349,7 @@ def vacuum(
     """
 
     dtw = DeltaTableWrapper(
-        path=path, version=None, columns=None, storage_options=storage_options
+        path=path, version=None, columns=None, storage_options=storage_options,
+        delta_storage_options=delta_storage_optoins,
```

Comment:
I think there is a small typo in `delta_storage_options` , once corrected Test cases will pass I guess

Reply:
Whoops, fixed!